packages/default/scss/scrollview/_layout.scss

```diff
@@ -146,19 +158,15 @@
 
         .k-scrollview-nav-wrap {
             position: absolute;
-            bottom: ($scrollview-pager-offset - ($scrollview-pager-item-spacing / 4));
-            left: ($scrollview-pager-item-spacing / 2);
-            right: ($scrollview-pager-item-spacing / 2);
+            bottom: 0;
+            left: 0;
+            right: 0;
             height: $scrollview-pager-height;
-            overflow: hidden;
 
             .k-scrollview-nav {
                 margin: 0;
-                padding: ($scrollview-pager-item-spacing / 4) 0 0 0;
                 display: block;
                 white-space: nowrap;
-                overflow-x: scroll;
-                overflow-y: hidden;
                 text-align: center;
```

Comment:
This change is breaking the current logic of scrolling the pager buttons. To check it, use the following demo https://demos.telerik.com/kendo-ui/scrollview/api